From d87b28ac4dac687d62580eaf2f34908e138e999e Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 5 Nov 2013 15:56:34 +0000
Subject: [PATCH] builder: Add dependency from index-parse.h to index-validate.

For some reason automake doesn't get this dependency right,
resulting in occasional build failures if yacc races the
rest of the build.

(cherry picked from commit e3ac293e869a4c69225e3d954b812e9bf21588a1)
---
 builder/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/builder/Makefile.am b/builder/Makefile.am
index 97550a8..3ad18cc 100644
--- a/builder/Makefile.am
+++ b/builder/Makefile.am
@@ -193,6 +193,8 @@ virt_index_validate_SOURCES = \
 	index-struct.c \
 	index-validate.c
 
+index-validate.o: index-parse.h
+
 man_MANS += virt-index-validate.1
 noinst_DATA += $(top_builddir)/html/virt-index-validate.1.html
 
-- 
1.8.3.1

